### DISCUSSION TITLE
unable to build project using Gradle build command

### DISCUSSION THREAD

**Discussion by RSapru** [Netflix/conductor#2463] [Q&A] [answered]
Hi, whenever I try yo build the project from root , I keep getting below error. I am trying to build the project in Windows 10 machine 
```
* What went wrong:
An exception occurred applying plugin request [id: 'nebula.netflixoss', version: '9.2.2']
> Failed to apply plugin class 'nebula.plugin.netflixossproject.publishing.PublishingPlugin'.
   > Task with name 'postRelease' not found in root project 'conductor'.
```

**Comment by RSapru** [comment, marked as answer]
I was trying to download the code as zip and then building it, turns out I was doing it wrong. When I downloaded it as GIT and build it worked just fine.

**Comment by anbxtres** [comment]
Hey @RSapru - We are huge fans of Conductor - let us know if you have any questions as you try things out and we will be happy to help & collaborate..
